Add cheatsheet links in supervision docs, readme

LinasKo opened the pull request:
# Description

Supervision cheatsheet was previously only posted on LinkedIn.

This change adds mention to it in supervision docs:
1. Readme
2. Nav bar (top bar)
3. Quickstart Cards at the bottom

## Type of change

-   [ ] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

mkdocs serve

## Any specific deployment considerations

## Docs

-   [x] Docs updated? What were the changes:

Added cheatsheet to:
1. Readme
2. Nav bar (top bar)
3. Quickstart Cards at the bottom

onuralpszr commented:
@LinasKo I can merge this but one question I couldn't find source code of cheatsheet ?

onuralpszr commented:
@LinasKo  First thing I notice some of the URL using "/develop" and some using "/latest/"  so it needs to be consistent to latest version not the develop

onuralpszr commented:
Suggestion: Every section almost has headline but not in here they all under one and "little mixed" looks maybe add extra h2/h3 headlines 

![image](https://github.com/roboflow/supervision/assets/1688848/5dd5fc01-90ec-444f-81b7-4b9257701ebf)

LinasKo commented:
Hey @onuralpszr 👋 

Good points!
1. Changed links to /latest
2. Removed duplicate number in footer
3. Did NOT add headers yet - perhaps in a future iteration.
4. Given that you're interested, I've open-sourced the repo: https://github.com/roboflow/cheatsheet-supervision

Note that it was rushed out with the tools I know, with close to 0 effort to make it readable. Might have some very rough edges :)

As for merging, I want to chat with @SkalskiP first.

onuralpszr commented:
> Hey @onuralpszr 👋
> 
> Good points!
> 
> 1. Changed links to /latest
> 2. Removed duplicate number in footer
> 3. Did NOT add headers yet - perhaps in a future iteration.
> 4. Given that you're interested, I've open-sourced the repo: https://github.com/roboflow/cheatsheet-supervision
> 
> Note that it was rushed out with the tools I know, with close to 0 effort to make it readable. Might have some very rough edges :)
> 
> As for merging, I want to chat with @SkalskiP first.

Is PRs Welcome to repo ? :)
